tests/test_prompt.py

```diff
@@ -260,6 +260,29 @@ def test_dont_prompt_for_private_context_var(self, monkeypatch):
         cookiecutter_dict = prompt.prompt_for_config(context)
         assert cookiecutter_dict == {'_copy_without_render': ['*.html']}
 
+    def test_render_hidden_variables(self):
```

Comment:
Docstrings missed.

Reply:
Yup, thanks. I haven't finished yet!

Reply:
I should have made it a draft PR actually. I'll do that.